### PR TITLE
Revert "[RIGA-2018] Adding new speaker"

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Reverts devopsdays/devopsdays-web#5195

There's a problem with the deploy I can't reproduce locally - reverting to see if I can narrow it down.

### CODE REVIEW

**Comment by mattstratton** [comment]
The problem is an error thrown during min-html. It seems to be related to the facebook pixel tracking code. You likely can't use javascript in a markdown file.

**Comment by mattstratton** [comment]
The other speaker files for Riga have the facebook javascript, so it might not be that simple.

**Comment by mattstratton** [comment]
For reference, this is the error in netlify:

```
9:24:05 AM: [16:24:04] Starting 'min-html'...
9:24:39 AM: events.js:137
9:24:39 AM:       throw er; // Unhandled 'error' event
9:24:39 AM:       ^
9:24:39 AM: Error: Parse Error: </!--><a href = "https://www.devopsdays.org/events/2018-riga/speakers/serhat-can/">...</a></span>
9:24:39 AM: </div>
9:24:39 AM: </div><div class="row cta-row">
```

**Comment by bridgetkromhout** [comment]
@mattstratton why this confused me is because that code is all over the place for Riga speakers, yet the others aren't breaking.

**Comment by mattstratton** [comment]
I just looked at the output HTML on another Riga page and it does show the javascript, so it seems to process.

File: content/events/2018-riga/speakers/serhat-can.md [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
- n.queue=[];t=b.createElement(e);t.async=!0;
- t.src=v;s=b.getElementsByTagName(e)[0];
- s.parentNode.insertBefore(t,s)}(window, document,'script',
- 'https://connect.facebook.net/en_US/fbevents.js');
- fbq('init', '627303307635674');
- fbq('track', 'PageView');
-</script>
-<noscript><img height="1" width="1" style="display:none"
- src="https://www.facebook.com/tr?id=627303307635674&ev=PageView&noscript=1"
-/></noscript>
```
Comment (mattstratton) [inline content/events/2018-riga/speakers/serhat-can.md]: Aha! This is the break.

Comment (mattstratton) [reply content/events/2018-riga/speakers/serhat-can.md]: look at how `/>` is on a separate line. That could be causing it.

Comment (bridgetkromhout) [reply content/events/2018-riga/speakers/serhat-can.md]: Ah! Wanna push the fix and I can close this PR?

Comment (mattstratton) [reply content/events/2018-riga/speakers/serhat-can.md]: Yeah, let me try that. Keep this open for now.

**Pull request merged**